CRM/ACL/API.php

```diff
@@ -65,6 +65,10 @@ public static function check($str, $contactID = NULL) {
       $contactID = 0;
     }
 
+    if ($allGroups == NULL) {
```

Comment:
@MegaphoneJon looking at the code around this, it doesn't seem to be in the right spot. didn't you mean to put this in the group function?